README.md

```diff
@@ -1,162 +1,174 @@
+* [Peter Hansteen](https://bsdly.blogspot.com)
+* [Joshua Stein](https://jcs.org/)
+* [Antoine Jacoutot](https://www.bsdfrog.org/)
+* [Adam Wołk](https://blog.tintagel.pl/)
+* [Nan Xiao](http://nanxiao.me/en/tag/openbsd/)
+* [Roman Zolotarev](https://www.romanzolotarev.com/openbsd/)
+* [Daniel Jakots](https://chown.me/)
+* [Solène Rapenne](https://dataswamp.org/~solene/)
+
+## News & forums
+
+* [Undeadly](http://undeadly.org/) - OpenBSD Journal
+* [Planet OpenBSD](http://openbsdnow.org/) - [src repo](https://github.com/ligurio/openbsdnow.org)
+* [Lobsters OpenBSD tag](https://lobste.rs/t/openbsd)
+* [reddit community](http://reddit.com/r/openbsd/)
+* [Google Plus community](https://plus.google.com/communities/113634135604793474364)
+* [UnitedBSD](https://unitedbsd.com/c/openbsd)
+
+## Mailing lists
+
+* [MARC](https://marc.info/):
+  * [openbsd-tech](http://marc.info/?l=openbsd-tech)
+  * [openbsd-cvs](http://marc.info/?l=openbsd-cvs)
+  * [openbsd-ports](http://marc.info/?l=openbsd-ports)
+  * [openbsd-misc](http://marc.info/?l=openbsd-misc)
+* [OpenBSD weekly/daily changes list](http://www.squish.net/log/openbsd-source-changes/)
+* [Mastering the web](https://www.romanzolotarev.com/n/) - Zolotarev's newsletter with lots of goodies
+
+## IRC
+
+* [#OpenBSD](https://webchat.freenode.net/#openbsd) (Freenode)
+
+## Twitter
+
+* [@OpenBSD](https://twitter.com/OpenBSD)
+* [@OpenBSD_CVS](https://twitter.com/OpenBSD_CVS) - OpenBSD Commit messages in 140 characters or less. For individual modules, see: [@OpenBSD_src](https://twitter.com/OpenBSD_src), [@OpenBSD_ports](https://twitter.com/OpenBSD_ports), [@OpenBSD_www](https://twitter.com/OpenBSD_www), [@OpenBSD_xenocar](https://twitter.com/OpenBSD_xenocar), [@OpenBSD_stable](https://twitter.com/OpenBSD_stable), [@OpenBSD_sets](https://twitter.com/OpenBSD_sets) (all built by [Andrew Fresh](https://twitter.com/afresh1))
+* [@OpenBSDJournal](https://twitter.com/openbsdjournal) - tweets to Undeadly.org stories
+* [@OpenSMTPD](https://twitter.com/opensmtpd)
+* [@OpenBSDNow](https://twitter.com/openbsdnow) - OpenBSD News & Updates
+* [@mpotd_openbsd](https://twitter.com/mpotd_openbsd) - Man Page of The Day
+* [@romanzolotarev](https://twitter.com/romanzolotarev) - lots of good advice
+
+## Mastodon ([bsd.network](https://bsd.network/) instance)
+
+* [@phessler](https://bsd.network/@phessler) - bsd.network's admin
+* [@AFresh1](https://bsd.network/@AFresh1)
+* [@bcallah](https://bsd.network/@bcallah)
+* [@romanzolotarev](https://bsd.network/@romanzolotarev)
+
+## Telegram Groups
+
+* [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8Mcj89LYcDGKiQ) from the [OpenBSD Jumpstart](http://www.openbsdjumpstart.org/) author
+* [OpenBSD Brazil](https://t.me/OpenBSDbr)
+* [OpenBSD Spain](https://t.me/OpenBSD_es)
+
+## Third Party repositories
+
+* [Scripts to run an OpenBSD mirror](https://github.com/bluhm/mirror-openbsd)
+* [OpenBSD work in progress ports](https://github.com/jasperla/openbsd-wip)
```

Comment:
There are many openbsd-wip repositories and japserla/openbsd-wip is only one of them. I suggest to add a link to a list of repositories with topic "openbsd-wip" - https://github.com/topics/openbsd-wip Repository owned by jasper is already there.

Reply:
Great! Good idea!